components/omega/doc/index.md

```diff
@@ -0,0 +1,46 @@
+(omega-home)=
+# OMEGA
+
+The Ocean Model for E3SM Global Applications (OMEGA) is an eddy-resolving, 
+global ocean model in the early stages of development by the 
+[E3SM](https://e3sm.org/) ocean team.  The first release is planned for
+Summer 2026.
+
+The model is written in c++ using the [YAKL](https://github.com/mrnorman/YAKL)
+framework for performance portability.  OMEGA is based on the 
+[TRSK formulation](https://doi.org/10.1016/j.jcp.2009.08.006) for geophysical 
+models on unstructured meshes. The first version of OMEGA will be a direct port
+of the [MPAS-Ocean](https://e3sm.org/model/e3sm-model-description/v1-description/v1-ocean-sea-ice-land-ice/)
+component of E3SM for comparison purposes.
+
+Development is taking place at https://github.com/E3SM-Project/Omega.
```

Comment:
@vanroekel, this is the main part I want your review on.  Would you like to flesh this out further (now or later)?  Anything here you're not happy with?

Reply:
Thanks for pointing me to this, I think it looks great.  I made a few minor tweaks.